.github/CODEOWNERS

```diff
@@ -13,10 +13,10 @@
 /automl/                        @nnegrey @telpirion @sirtorry @GoogleCloudPlatform/python-samples-owners
 /bigquery/                      @shollyman @GoogleCloudPlatform/python-samples-owners
 /bigquery_storage/              @shollyman @GoogleCloudPlatform/python-samples-owners
-/bigtable/                      @billyjacobson @GoogleCloudPlatform/python-samples-owners
+/bigtable/                      @GoogleCloudPlatform/bigtable-dpe @GoogleCloudPlatform/python-samples-owners
 /blog/                          @GoogleCloudPlatform/python-samples-owners
 /cdn/                           @mpwarres @elithrar @GoogleCloudPlatform/python-samples-owners
-/cloud-sql/                     @kvg @GoogleCloudPlatform/python-samples-owners
+/cloud-sql/                     @GoogleCloudPlatform/sql-dpe @GoogleCloudPlatform/python-samples-owners
```

Comment:
```suggestion
/cloud-sql/                     @GoogleCloudPlatform/cloud-sql-dpes @GoogleCloudPlatform/python-samples-owners
```

Reply:
The team name is currently @GoogleCloudPlatform/sql-dpe. Want me to change the team name and then change throughout?